.buildkite/pipeline.yaml

```diff
@@ -80,6 +82,13 @@ steps:
     command: .buildkite/build-emscripten.sh
     artifact_paths: _out_/**/*
     <<: *elastic
+
+  - label: ":mac: build-vscode-extension.sh"
```

Comment:
It looks like I might have missed updating this to run on the linux agent :( 

Honestly we might not even need this right now until automatic publishing (I believe this was my intention originally). I'd recommend pulling it out, as the test should build it.

Reply:
Done.